charts/scorpio-broker-dist/templates/eureka-node-port.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.eureka.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: {{ .Values.eureka.eurekaLabel }}
+  name: {{ include "eureka.fullname" . }}-node-port
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.eureka.service.type }}
+  ports:
+  - port: {{ .Values.eureka.service.port }}
+    targetPort: {{ .Values.eureka.service.port }}
```

Comment:
The nodeport should only be enabled if the service is of type nodeport

Reply:
We have incorporated the review comment. Please review again